CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contribution guidelines
+
+We love and welcome contributions!
+
+In order to maintain a high quality, we run a great number of checks on
+different OS / Compiler combinations, and using different analysis tools.
+
+## Prerequisites
+
+Building and testing `sentry-native` currently requires the following tools:
+
+- **CMake** and a supported C/C++ compiler, to actually build the code.
+- **python** and **pytest**, to run integration tests.
+- **clang-format** and **black**, to format the C/C++ and python code respectively.
+
+## Setting up Git Hooks
+
+    $ make setup
+
+Installs a pre-commit hook, and initializes/updates the necessary git submodules.
+
+## Formatting Code
+
+    $ make format
+
+This should be done automatically as part of the pre-commit hook, but can also
+done manually.
+
+    $ black tests
```

Comment:
That should be `make format-python`.

Reply:
We don’t have separate format targets. Also since I still haven’t managed to properly set up make/bash on windows, I tend to invoke `black` manually there. I also invoke pytest there manually. So I would like to give the commands without using any make/bash specifics.